types/bootstrap.v3.datetimepicker/index.d.ts

```diff
@@ -2,8 +2,8 @@
 // Project: http://eonasdan.github.io/bootstrap-datetimepicker
 // Definitions by: Katona Péter <https://github.com/katonap>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
 // based on the previous version created by Jesica N. Fera <https://github.com/bayitajesi>
+// TypeScript Version: 2.3
```

Comment:
I'm a little worried about this change because I'm not sure tooling can handle it being separated from the header like this. I'm assuming `types-publisher` at least handles this correctly since tests passed.